### PR TITLE
[MUS-57] music player component

### PR DESCRIPTION
**Pull request by Drelarusoidd**

### CODE REVIEW

**Review by nickkuchko — APPROVED** [review]
Everything looks good

File: src/App.css [hunk 1]
```diff
@@ -18,4 +18,137 @@
...
+    z-index: 2;
+}
+
+.audio-player .volume-percent {
+    background-color: #f50;
+    border-bottom: 2px solid;
+    height: 100%;
+    width: 100%;
+    z-index: 3;
+}
```
Comment (ZubelRuslan) [inline src/App.css:154]: Pls move to separate file audio player styles

File: src/components/PlayerComponent.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            current_seconds_long = currentTime % 60,
+            current_seconds = current_seconds_long.toFixed(),
+            current_time =
+                (current_minute < 10 ? "0" + current_minute : current_minute) +
+                ":" +
+                (current_seconds < 10 ? "0" + current_seconds : current_seconds);
+      
+        return current_time;
+    }
+
```
Comment (ZubelRuslan) [inline src/components/PlayerComponent.js:46]: Looks like you can move this functions to another utils file

File: src/components/PlayerComponent.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    const initProgressBar = (num) => {
+        var player = document.getElementById(`player-${num.index}`);
+        var length = player.duration;
+        var current_time = player.currentTime;
+    
+        var totalLength = calculateTotalValue(length);
+        $(`.end-time-${num.index}`).html(totalLength);
```
Comment (ZubelRuslan) [inline src/components/PlayerComponent.js:53]: remove jquery. You can use React refs

**Review by ZubelRuslan — CHANGES_REQUESTED** [review]
I see a lot of copy pasted NOT REACT code. Change it!

**Comment by ZubelRuslan** [comment]
> Everything looks good
m? what about player component?

**Pull request merged**